python/pyiceberg/catalog/rest.py

```diff
@@ -113,7 +113,7 @@ class Endpoints:
 
 class TableResponse(IcebergBaseModel):
     metadata_location: str = Field(alias="metadata-location")
-    metadata: TableMetadata = Field()
```

Comment:
Did you remove the `= Field()` for a specific reason?

Reply:
After using Annotated, I got this error
```ValueError: cannot specify `Annotated` and value `Field`s together for 'metadata'```
So, I remove it

Reply:
Ah makes sense, thanks!